### DISCUSSION TITLE
I claimed wrong shimmer staking rewards i claimed only 148 instead of alot more :(

### DISCUSSION THREAD

**Discussion by Vlad100000** [iotaledger/firefly#7914] [Help & Support]
![shimmer claimed wrong amount](https://github.com/iotaledger/firefly/assets/157259612/e6f280ae-d099-40d0-9d45-18e82a3fd482)

**Comment by Vlad100000** [comment]
hi guys so i claimed a wrong amount of my shimmer and if i click on search again he finds nothing anymore ...
so idk what to do right now ,
can any one help me out ?

**Reply by begonaalvarezd** [reply]
If you believe you have more, you can keep searching, if your do have more rewards, they will appear.

In any case, please note there was a unit mismatch in firefly which was corrected a bit later: `The new Firefly release updates the SMR denominations to match the correct formatting. 1 SMR now has 6 decimal places. You have the same amount of SMR rewards, they're just displayed correctly now`
